src/Template/Pages/Model/index.twig

```diff
@@ -0,0 +1,75 @@
+{% do _view.assign('title', 'Model ' ~ resourceType|humanize) %}
+{% do _view.assign('bodyViewClass', 'view-model') %}
+
+{% set ids = Array.extract(resources, '{*}.id') %}
+
+{% set resourceType = _view.request.params.resource_type %}
+
+<modules-index inline-template ids='{{ ids|json_encode }}'>
+    <div class="model-index">
+        <div class="list-resources">
+
+            {# list resources : table header #}
+            {% if resources %}
+            <nav class="table-header">
+                <div @click="toggleAll">
+                    <input type="checkbox" :checked="allChecked">
+                </div>
+
+                <div>{{ Link.sort('id')}}</div>
+
+                {% for prop in properties %}
+                    <div>{{ Link.sort(prop) }}</div>
+                {% endfor %}
+                <div></div>
+
+            </nav>
+            {% endif %}
+
+            {# list resources : rows #}
+            {% for resource in resources %}
+                <a
+                    href="{{ Url.build({'_name': 'model:view', 'resource_type': resource.type, 'id': resource.id}) }}"
+                    title="{{ resource.attributes.name }}"
+                    class="resource-status-on">
+
+                        <div class="{{ prop }}-cell" @click="selectRow">
+                            {{ Form.unlockField('oneItem') }}
+                            <input type="checkbox" name="oneItem" value="{{ resource.id }}" v-model="selectedRows">
+                        </div>
+
+                        <div>{{ resource.id }}</div>
+
+                        {% for prop in properties %}
+                            {% set propVal = resource.attributes[prop] %}
+                            {% if propVal is iterable %}
+                                {% set propVal = propVal|json_encode(constant('JSON_PRETTY_PRINT')) %}
+                            {% elseif  propVal is same as(true) %}
+                                {% set propVal = 'Y' %}
+                            {% elseif  propVal is same as(false) %}
+                                {% set propVal = 'N' %}
+                            {% endif %}
+
+                            <div class="{{ prop }}-cell" untitled="{{ __('Untitled') }}">{{ propVal }}</div>
+                        {% endfor %}
+
+                        {# <div class="buttons">
```

Comment:
Is this comment here as a reminder? If yes I will write some note about why it is commented and/or what missing to make it works properly.

Reply:
Reminder to add buttons/actions to delete or change resources, not yet implemented... will add a comment